### DISCUSSION TITLE
Support extra parameters on datasource uri

### DISCUSSION THREAD

**Discussion by tchatzig** [burningalchemist/sql_exporter#627] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Currently we are running sql-exporter on our stacks and it works fine for us. 
However, when we check the Performance Insights on the AWS side we cannot get the data we want.

**Describe the solution you'd like**
It would be nice if there was a way to add `?application_name=` on the datasource uri likes its done here for example:
https://github.com/prometheus-community/helm-charts/commit/ed35464b840173a39bb1be170bd85211170a8290

Is there a way to do that with the current helm chart?

**Comment by burningalchemist** [comment, marked as answer]
Hi @tchatzig, you can just add the parameters to the data_source_name directly:

```
 data_source_name: 'pg://test@127.0.0.1:5432/db?sslmode=disable&application_name=myapp'
```

I think it should work just fine. Make sure that the parameter is supported by the driver to be interpreted correctly.

Also if the driver supports any reserved environment variables, you can also add them to the container envs (e.g. `PGAPPNAME=myapp`).

**Comment by burningalchemist** [comment]
I'll convert this ticket to discussion at the moment. :)
